Makefile

```diff
@@ -79,9 +79,19 @@ generate-ts-doc: ## This command generates documentation from the typescript fil
 test-e2e: ## This command executes PlayWright E2E tests.
 	$(PLAYWRIGHT_EXEC) test:e2e
 
-test-visual: ## This command executes PlayWright Visual tests.
+start-prod: ##  Build image and start container in production mode
+	$(DOCKER_COMPOSE) -f docker-compose.test.yml up -d
+
+playwright-visual: ## This command executes Playwright visual tests.
 	$(PLAYWRIGHT_EXEC) test:visual
 
+test-visual: start-prod playwright-visual  ## Start production, wait for it to be ready, and run Playwright visual tests.
```

Comment:
It will be better to add wait-for-prod after the start-prod and before playwright-visual